docs/conf.py

```diff
@@ -169,6 +169,7 @@
         "locations.html",
         "archives.html",
     ],
+    "index": [],
```

Comment:
-1 on this change.  The theme is supposed to suppress the sidebars automatically when they don't have any content.  Doing this makes our site look right, but hides the fact that the theme isn't doing what it's supposed to do. I'd prefer we fix the behavior so that the empty sidebar is properly suppressed.